nb/projects/plotter/plotter.stars_0.gcode

```diff
@@ -2,377 +2,270 @@
 G21
```

Comment:
We might want to add a G90 or G91 to the front of these to indicate absolute vs incremental moves. G90 is absolute.

Reply:
Ah, good point.